persistence/src/test/scala/hmda/persistence/institutions/InstitutionPersistenceSpec.scala

```diff
@@ -34,6 +34,24 @@ class InstitutionPersistenceSpec extends ActorSpec {
       probe.send(institutionsActor, GetInstitutionById(modified.id.toString))
       probe.expectMsg(modified)
     }
+
+    "return a set of institutions matching a list of ids" in {
+      // Setup: Persist demo institutions
+      val i1 = Institution(71, "CFPBank", Set(ExternalId("99-1234567", FederalTaxId)), CFPB, Bank, hasParent = true, Active)
+      val i2 = Institution(72, "FRBank", Set(ExternalId("654321", RssdId)), CFPB, Bank, hasParent = true, Active)
+      val i3 = Institution(73, "MLBank", Set(ExternalId("externalTest0", FdicCertNo)), FDIC, Bank, hasParent = true, Active)
+      for (institution <- List(i1, i2, i3)) {
+        probe.send(institutionsActor, CreateInstitution(institution))
```

Comment:
OH YEAH. So I just tried your suggestion (good suggestion, by the way!), and I remembered the real reason why I had to create my own data with new ids. In the collection of persisted Institutions, there are already multiple records with id 0, 1, 2, and 3. So when I use `GetInstitutionsById`, I get multiple responses for each `id`. The fact that we allow duplicate IDs is a separate issue, so I'll create a GH issue for it.